vortex-serde/src/layout/mod.rs

```diff
@@ -0,0 +1,267 @@
+use std::fmt::Debug;
+use std::future::Future;
+use std::pin::Pin;
+use std::sync::Arc;
+use std::task::{Context, Poll};
+
+use bytes::Bytes;
+use hashbrown::HashMap;
```

Comment:
i thought as of a few years ago std is just hashbrown

Reply:
Mostly, I can use std just need to override hashing algorithm

Reply:
ah ok yea hash fns makes sense 👍